rabbitmq/tests/metrics.py

```diff
@@ -39,6 +39,7 @@
 
 Q_METRICS = [
     'rabbitmq.queue.consumers',
+    'rabbitmq.queue.consumer_utilisation',
```

Comment:
You'll have to add this one only for version >= 3.8, i.e

```python
# Present from 3.8
if RABBITMQ_VERSION >= version.parse('3.8'):
    Q_METRICS.extend(['rabbitmq.queue.consumer_utilisation'])
```

Reply:
FIxed. I'm waiting CI.. 😅 

Thank you for review.